xsd/netex-nl-enums.xsd

```diff
@@ -599,32 +599,86 @@
 	<!-- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -->
 	<xsd:complexType name="typeOfService">
 		<xsd:complexContent>
-			<xsd:extension base="TypeOfValueVersionStructure"/>
+			<xsd:extension base="TypeOfValueVersionStructure">
+				<xsd:attribute name="version" type="VersionIdType" use="required" fixed="any" form="unqualified">
+					<xsd:annotation>
+						<xsd:documentation>
+							Het 'version' attribuut is nodig voor de constraints, maar heeft in voorgedefinieerde elementen altijd de vaste waarde 'any'.
```

Comment:
We hebben in de Europese XSD het ding dat we van commentaar 1 regel maken, zodat wijzigingen altijd direct zichtbaar zijn.